### PR TITLE
For_VBFGammaH_HToBB_Run2_study

### PR DESCRIPTION
**Pull request by Soumyatifr**

### CODE REVIEW

**Comment by agrohsje** [comment]
Hi @Soumyatifr . Why is ickkw = 1 ?

**Comment by Soumyatifr** [comment]
Dear Experts, 
It was my mistake. 
I have set  ickkw = 0 as recommended, for all the three run_cards. Please check if there is any further issue.
Thanks a lot

**Comment by agrohsje** [comment]
Note that the scripts will add True = use_syst to the run_card. Usually people have the setting there per default. It is needed for systematics computations.  
import model sm, will give you 4FS not 5FS. maxjetflavor is set to 5.

**Comment by Soumyatifr** [comment]
(i) One line added in the bottom of the run card "T = use_syst".  
(ii) maxjetflav set to 4. as "import model sm"

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBF_Zgamma_ZTobb_Mjj_200/VBF_Zgamma_ZTobb_Mjj_200_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+#*********************************************************************
+   False  = cut_decays    ! Cut decay products 
+#*********************************************************************
+# Standard Cuts                                                      *
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+ 20.0  = ptj       ! minimum pt for the jets 
+ 20.0  = ptb       ! minimum pt for the b 
+ 25.0  = pta       ! minimum pt for the photons
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBF_Zgamma_ZTobb_Mjj_200/VBF_Zgamma_ZTobb_Mjj_200_run_card.dat:112]: Note that you are applying these cuts? Is 25 GeV well above the cut at detector level? Same for eta of 2.5? What are the final cuts.

**Comment by agrohsje** [comment]
Dear @Soumyatifr . Please check that the cuts are loose enough for the final selection.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBF_Zgamma_ZTobb_Mjj_200/VBF_Zgamma_ZTobb_Mjj_200_run_card.dat (hunk 1 above)
Comment (Soumyatifr) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBF_Zgamma_ZTobb_Mjj_200/VBF_Zgamma_ZTobb_Mjj_200_run_card.dat]: We are planing to apply offline cuts for hard photon is > 25/50 GeV inside the |eta| < 2.5 . That's why we are keeping the cuts of the gen level photons to these. But now UI have changed it to > 20 GeV.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBF_Zgamma_ZTobb_Mjj_200/VBF_Zgamma_ZTobb_Mjj_200_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+  -1.0   = ejmax ! maximum E for the jets
+ -1.0   = ebmax ! maximum E for the b
+ -1.0   = eamax ! maximum E for the photons
+ -1.0   = elmax ! maximum E for the charged leptons
+#*********************************************************************
+# Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
+#*********************************************************************
+  5.0 = etaj    ! max rap for the jets 
+  -1.0  = etab    ! max rap for the b
+ 2.5  = etaa    ! max rap for the photons
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBF_Zgamma_ZTobb_Mjj_200/VBF_Zgamma_ZTobb_Mjj_200_run_card.dat:136]: Wait. But this is the same at gen and detector level. Wouldn't it make sense to use a looser cut.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBF_Zgamma_ZTobb_Mjj_200/VBF_Zgamma_ZTobb_Mjj_200_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+ 0.4 = drjj    ! min distance between jets 
+ 0.0   = drbb    ! min distance between b's 
+ 0.4 = drll    ! min distance between leptons 
+ 0.4 = draa    ! min distance between gammas 
+ 0.0   = drbj    ! min distance between b and jet 
+ 0.4 = draj    ! min distance between gamma and jet 
+ 0.0 = drjl    ! min distance between jet and lepton 
+ 0.4   = drab    ! min distance between gamma and b 
+ 0.0   = drbl    ! min distance between b and lepton 
+ 0.4 = dral    ! min distance between gamma and lepton
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBF_Zgamma_ZTobb_Mjj_200/VBF_Zgamma_ZTobb_Mjj_200_run_card.dat:154]: For leptons this is irrelevant, but why are the cuts different for b jets and light jets? Is this on purpose?

Comment (Soumyatifr) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBF_Zgamma_ZTobb_Mjj_200/VBF_Zgamma_ZTobb_Mjj_200_run_card.dat:154]: yes delta R for leptons are not usefull so should I remove the lines corresponding to the leptons. But for b-jets and q-jets which cuts are different I mean I imposed same deltaR cuts between the photon and b-jets and also between photon and q-jets. I got the clue from here: https://github.com/Soumyatifr/genproductions/blob/mg260legacy/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBF_HH/VBF_HH_CV_1_C2V_1_C3_1_13TeV-madgraph/VBF_HH_CV_1_C2V_1_C3_1_13TeV-madgraph_run_card.dat#L143-L163 .
Please let me know if I missed something.

Comment (Soumyatifr) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBF_Zgamma_ZTobb_Mjj_200/VBF_Zgamma_ZTobb_Mjj_200_run_card.dat:154]: @agrohsje please check if you have further comments.

**Pull request merged**